tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/worker/BasicForkNegativeTest.java

```diff
@@ -39,10 +39,20 @@ public void setup() {
 
     @Test
     public void testBasicForkNegative() {
-        Assert.assertEquals(result.getErrorCount(), 2, "Incorrect error count");
-        BAssertUtil.validateError(result, 0,
```

Comment:
Do we have a similar test for between workers in a fork statement?

Reply:
Issue created to address this https://github.com/ballerina-platform/ballerina-lang/issues/18430